src/compiler/scala/tools/nsc/Global.scala

```diff
@@ -1602,8 +1602,11 @@ class Global(var currentSettings: Settings, reporter0: LegacyReporter)
         val snap = profiler.beforePhase(Global.InitPhase)
 
         val sources: List[SourceFile] =
-          if (settings.script.isSetByUser && filenames.size > 1) returning(Nil)(_ => globalError("can only compile one script at a time"))
-          else filenames map getSourceFile
+          if (settings.script.isSetByUser && filenames.size > 1) {
+            globalError("can only compile one script at a time")
+            Nil
+          }
+          else filenames.map(getSourceFile)
```

Comment:
What did `returning` ever do to you?

Reply:
Poor return on investment of time in reading that line.